index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile Skeleton</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="logo-sec">
+      <a href="#" class="wel">Welcome</a>
+      <div class="menu">
+        <img
+          class="image"
+          src="union-1/Normal Button/Tertiary/Icons/Union.svg"
+          alt="menu"
+        />
+      </div>
+    </header>
+
+    <section class="info1">
+      <div class="heading">
+        <h1 class="heading1">Hello There I'm Theodore</h1>
+        <h2 class="heading2">I am a Software Developer</h2>
+      </div>
+
+      <div class="texting">
+        <p class="text1">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+      </div>
+
+      <div class="icons">
+        <div class="icon1">
+          <img src="twitter.svg" alt="twitter" />
+        </div>
+        <div class="icon2">
+          <img src="Linkedin icon.svg" alt="Linkedin" />
+        </div>
+        <div class="icon3">
+          <img src="medium.svg" alt="medium" />
+        </div>
+        <div class="icon4">
+          <img src="github.svg" alt="github" />
+        </div>
+        <div class="icon5">
+          <img src="angellist.svg" alt="angelist" />
+        </div>
+      </div>
```

Comment:
- [ ] Let's change all of the `divs` in this block for an Unordered List and Lists Items. For this, we will use an `ul` tag to wrap the items of the list, and `li` tags to wrap the inner items.